packages/playwright/src/isomorphic/teleReceiver.ts

```diff
@@ -512,8 +516,11 @@ class TeleTestStep implements reporterTypes.TestStep {
   parent: reporterTypes.TestStep | undefined;
   duration: number = -1;
   steps: reporterTypes.TestStep[] = [];
+  attachments: reporterTypes.TestStep['attachments'] = [];
+  error?: reporterTypes.TestError | undefined;
```

Comment:
Drive-by?

Reply:
it's a side-effect of the change in line 560. I'm changing the type there to get access to `_attachmentIndices` in L270. The type change also makes L269 complain that there's no error field.